onnxruntime-sys/build.rs

```diff
@@ -233,6 +261,17 @@ impl OnnxPrebuiltArchive for Architecture {
     }
 }
 
+impl Architecture {
+    fn as_onnx_directml_str(&self) -> Cow<str> {
+        match self {
+            Architecture::X86 => Cow::from("x86"),
+            Architecture::X86_64 => Cow::from("x64"),
+            Architecture::Arm => Cow::from("arm"),
+            Architecture::Arm64 => Cow::from("aarch64"),
```

Comment:
すみません、細かいのですが arm64 の場合もディレクトリ名が `win-arm64` なので変更が必要です。

```suggestion
            Architecture::Arm64 => Cow::from("arm64"),
```

Reply:
修正しました